backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.test.ts

```diff
@@ -391,181 +390,201 @@ describe("Async Issue Biometric Credential", () => {
         it("Does not log COMPLETED", () => {
           expect(consoleInfoSpy).not.toHaveBeenCalledWithLogFields({
             messageCode: "MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_COMPLETED",
-            sessionId: mockSessionId,
           });
         });
       });
     });
   });
 
-  describe("When there is an error getting secrets", () => {
+  describe("When biometric session state is RESULT_SENT", () => {
```

Comment:
```suggestion
  describe("When session state is ASYNC_RESULT_SENT", () => {
```

This is the user session, not the biometric session.

As a reminder:
* the user session is stored in dynamo, managed by us, has a bunch of useful fields in (e.g. subject ID, client ID, redirect URI), and follows a state machine (e.g. ASYNC_AUTH_SESSION_CREATED, ASYNC_BIOMETRIC_TOKEN_ISSUED, etc)
* the biometric session is managed by ReadID. The mobile app started the biometric session with ReadID and we're retrieving it in this ticket. It will include results of the checks that ReadID has done.

Reply:
comitted this